macros/src/main/scala-2.11+/pureconfig/Derivation.scala

```diff
@@ -0,0 +1,196 @@
+package pureconfig
+
+import scala.collection.mutable
+import scala.language.experimental.macros
+import scala.reflect.macros.whitebox
+
+import pureconfig.derivation._
+
+sealed trait Derivation[A] {
+  def value: A
+}
```

Comment:
This comment is really not important but I'm curious: why `Derivation` is not in a shared package that is then used by all the scala versions?